clang/test/SemaCXX/constexpr-string.cpp

```diff
@@ -670,6 +670,8 @@ namespace MemcpyEtc {
   constexpr bool test_address_of_incomplete_struct_type() { // expected-error {{never produces a constant}}
     struct Incomplete;
     extern Incomplete x, y;
+    // expected-warning@+2 {{destination for call to '__builtin_memcpy' is a pointer to non-trivially copyable type 'Incomplete'}}
```

Comment:
"destination for call" sounds a bit strange to me when reading. I'd write "destination argument/parameter for call" 

But we are splitting hairs at this point, if everyone else are happy with the current message I'm ok.